### ISSUE TITLE
System.Runtime.CompilerServices.Unsafe

### ISSUE THREAD

**Issue by PandaWood** [Tydude4Christ/Enums.NET#42]
No doubt this is my fault, with some other config
But I had to roll back Enums.NET v3 to avoid getting this runtime error - only when running tests.

```
Could not load file or assembly 'System.Runtime.CompilerServices.Unsafe, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at System.Runtime.CompilerServices.UnsafeUtility.As[TFrom,TTo](TFrom& source)
   at EnumsNET.Enums.GetName[TEnum](TEnum value) in C:\Users\tydud\source\repos\TylerBrinkley\Enums.NET\Src\Enums.NET\Enums.cs:line 963
```

**Comment by cvasquez-github** [comment]
Similar issue here, could it be updated to use the latest stable System.Runtime.CompilerServices.Unsafe 6.0?

**Comment by FelipeCostaGualberto** [comment]
Same here, I have to use now version 3.0.3 to prevent this error.

**Comment by softboy99** [comment]
same problem

**Comment by softboy99** [comment]
we use npoi ,which depends on this >=4.0.1 version, pls help repack nuget pack,thanks

**Comment by TylerBrinkley** [comment]
Fixed in #49
